docker-compose.dev.yml

```diff
@@ -72,4 +74,14 @@ services:
       EMAIL_TO: "subsidiepunt-development@redpencil.io"
     restart: "no"
   mock-account-generation:
-    restart: "no"
\ No newline at end of file
+    restart: "no"
+  ldes-delta-pusher:
+    restart: "no"
+    environment:
+      DEBUG: "true"
+      LDES_BASE: "http://localhost:90/streams/ldes/subsidies"
+  ldes-backend:
+    restart: "no"
+    environment:
+      BASE_URL: "http://localhost:90/streams/ldes/subsidies/"
+      LDES_STREAM_PREFIX: "http://localhost:90/streams/ldes/subsidies/"
```

Comment:
Note, these variables often need to be updated depending on the deploy-environment.
Also note if you want to check the sync between stacks, you will need to update the host names.